docs/index.adoc

```diff
@@ -264,15 +349,16 @@ If the FMU contains controller code, numeric stability or energy preservation la
 On the other hand, plant models offering XCP access for parameter calibration may introduce surprising numerical effects in solvers that might require proper handling, like resetting solvers with every XCP write action.
 
 It is therefore necessary to synchronize XCP variable access (read and write) with the state of the FMU.
-_[For instance is time not linear in Model Exchange and Intermediate Variable Access might also introduce surprising measurements in Co-Simulation._
+_[For instance is time not necessarily equidistant and not necessarily monotonically increasing in Model Exchange, and Intermediate Variable Access might also introduce surprising effects in Co-Simulation._
+_This means that some kind of buffering is necessary._

```

Comment:
I wouldn't add this sentence. Buffering is just one possible mechanism and the normative text mentions already that some kind of synchronization is necessary.